_pages/general/visas.md

```diff
@@ -5,17 +5,89 @@ description: How to obtain a visa, useful resources and FAQ's relating to visas
 
 # Visas
 
-If you require a working visa, you'll need to ensure you satisfy the requirements set out by the Australian immigration
-department before you commence our recruitment process. Click [here](https://www.homeaffairs.gov.au/trav/visa-1/482-){:target="\_blank"} to find the most current information relating to the
-482 visa.
+If you require a working visa, you'll need to ensure you satisfy the requirements set out by the Australian immigration department before you commence a recruitment process. Click [here](https://www.homeaffairs.gov.au/trav/visa-1/482-){:target="\_blank"} to find the most current information relating to the 482 visa.
 Once your visa is approved, you can get started setting up things you'll need for life in Australia. For more
-information on 482 visas, please visit the FAQs section.
+information on 482 visas.
 
-## How long does the application process take?
+## FAQ's
+
+### What type of visa should I apply for?
+
+**Skilled Independent (Subclass 189)**
+
+If you are a qualified professional or trades person with work experience that will benefit the Australian economy, and your occupation is on Medium and Long-term Strategic Skills List (MLTSSL), this visa is for you.
```

Comment:
is it worth having a link here?

Reply:
Eligibility link https://immi.homeaffairs.gov.au/visas/getting-a-visa/visa-listing/skilled-independent-189/points-tested#Eligibility

Reply:
Updated accordingly